.NET/Microsoft.Recognizers.Text.NumberWithUnit/Constants.cs

```diff
@@ -14,5 +14,9 @@ public static class Constants
         public const string SYS_UNIT_TEMPERATURE = "builtin.unit.temperature";
         public const string SYS_UNIT_VOLUME = "builtin.unit.volume";
         public const string SYS_UNIT_WEIGHT = "builtin.unit.weight";
+        public const string SYS_UNIT_COMPOUND = "builtin.unit.compound";
+        public const string SYS_NUM = "builtin.num";
+
+        public const string FAKE_ISO_CODE_PREFIX = "_";

```

Comment:
Add a code comment here, like: "// For currencies without ISO codes, we use internal values prefixed by '_'. This values should never be present in parse output."